### PR TITLE
Djia/sanitize

### PR DESCRIPTION
**Pull request by djia-vm-wf**

### CODE REVIEW

File: senders/formatter.go [hunk 1]
```diff
@@ -308,3 +314,52 @@ func isUUIDFormat(str string) bool {
 	}
 	return true
 }
+
+func sanitizeInternal(str string) string {
```
Comment (vikramraman) [inline senders/formatter.go]: Could you add some comments to this method and `sanitizeValue` describing what it does? It'd be useful to someone looking at this independently to understand what we're doing.

File: senders/formatter.go [hunk 2]
```diff
@@ -308,3 +314,52 @@ func isUUIDFormat(str string) bool {
...
+			sb.WriteString("-")
+		}
+	}
+	return sb.String()
+}
+
+func sanitizeValue(str string) string {
+	res := strings.TrimSpace(str)
+	if strings.Contains(str, "\"") || strings.Contains(str, "'") {
+		resReg, _ := regexp.Compile("\"")
```
Comment (vikramraman) [inline senders/formatter.go]: Make these global vars and pre-compile the regular expression. You can then reuse it within this method. Compiling it for every call is expensive and should be avoided.

Use `MustCompile(...)` for initializing the global vars.

File: senders/formatter.go [hunk 3]
```diff
@@ -308,3 +314,52 @@ func isUUIDFormat(str string) bool {
...
+		res = resReg.ReplaceAllString(res, "\\\"")
+	}
+	resReg, _ := regexp.Compile("\\n")
```
Comment (vikramraman) [inline senders/formatter.go]: Ditto. Make it a global var.

File: senders/formatter.go (hunk 2 above)
Comment (djia-vm-wf) [reply senders/formatter.go]: these two can actually be const. made them const

**Pull request merged**